pkgs/applications/video/tanidvr/default.nix

```
@@ -0,0 +1,19 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "tanidvr";
+  version = "1.4.1";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/tanidvr/TaniDVR/${pname}-${version}/${pname}-${version}.tar.bz2";
+    sha256 = "0irwwf6mb72n3y4xcrl3s081nbnldvdlc6ypjqxa4p32c1d0g6ql";
+  };
+
+  meta = {
+    description = "TaniDVR is a CLI tool for managing and capturing video from Linux-based OEM surveillance DVRs which use DVR-IP protocol";
+    homepage = "https://sourceforge.net/projects/tanidvr";
+    license = lib.licenses.gpl3;
+    maintainers = [ lib.maintainers.pho ];
```

Nit, feel free to ignore:

```suggestion
    maintainers = with lib.maintainers; [ pho ];
```